lib/puppet/provider/sensu_oidc_auth/sensuctl.rb

```diff
@@ -0,0 +1,128 @@
+require File.expand_path(File.join(File.dirname(__FILE__), '..', 'sensuctl'))
+
+Puppet::Type.type(:sensu_oidc_auth).provide(:sensuctl, :parent => Puppet::Provider::Sensuctl) do
+  desc "Provider sensu_oidc_auth using sensuctl"
+
+  mk_resource_methods
+
+  def self.instances
+    auths = []
+
+    data = sensuctl_list('auth', false)
+
+    auth_types = sensuctl_auth_types()
+    data.each do |d|
+      auth = {}
+      auth[:ensure] = :present
+      auth[:name] = d['metadata']['name']
+      if auth_types[auth[:name]] != 'OIDC'
+        next
+      end
+      #auth[:client_id] = d['client_id']
+      #auth[:client_secret] = d['client_secret']
```

Comment:
why the commented out code?

Reply:
That was leftover from development, I removed commented out code.